packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -320,27 +321,33 @@ export class ProjectLifecycleManager {
       return
     }
 
-    if (!this.metaState.hasValidConfigFile) {
-      if (testingType === 'e2e' && !this.ctx.isRunMode) {
-        this.ctx.actions.wizard.scaffoldTestingType().catch(this.onLoadError)
```

Comment:
I assume that we need to add in a graceful error message for users that hit this when in run mode.